src/Pages/ProductList.css

```diff
@@ -0,0 +1,78 @@
+.productListContainer{
+    display: grid;
+    grid-template-columns: 1fr 5fr;
+    margin-top: 4rem;
+    min-height: 80vh;
+}
+
+.productLeftContainer{
+    padding: 0 1rem;
+}
+.productLeftContainer ul{
+    list-style: none;
+}
+.productLeftContainer li{
+    list-style: none;
+    margin-left: -2rem;
+}
+.productLeftContainer ul:hover{
+    cursor: pointer;
+}
+.productRightContainer{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    padding: 0 1rem;
+    position: relative;
+}
+.spinner{
+    position: absolute;
+    left: 40%;
+}
+.ecommerceCard{
+    background-color: white;
+    position: relative;
+}
+.outOfStockCard{
+    position: absolute;
```

Comment:
Its good to avoid writing class names in camel case. Use hyphens, underscores...